### ISSUE TITLE
podman: option parsing failed: Unknown option --log-level

### ISSUE THREAD

**Issue by aalba6675** [projectatomic/libpod#1260]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
>
> /kind feature
/kind bug

**Description**
On master branch: podman cannot start containers. (Note: podman-0.8.1-1.git6b4ab2a.fc28.x86_64
works)

**Steps to reproduce the issue:**

1. build podman on master
[root@podhost centos]# /usr/local/bin/podman version
Version:       0.8.2.1
Go Version:    go1.10.3
OS/Arch:       linux/amd64

2.

3.

**Describe the results you received:**
```
[root@podhost centos]# /usr/local/bin/podman run --rm -it busybox echo hello world
option parsing failed: Unknown option --log-level
```

**Describe the results you expected:**
container starts up

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
[root@podhost centos]# /usr/local/bin/podman version
Version:       0.8.2.1
Go Version:    go1.10.3
OS/Arch:       linux/amd64```
```

**Output of `podman info`:**

```
# /usr/local/bin/podman info
host:                                                                                                                                                    
  Conmon:
    package: conmon-1.11.1-1.git1759204.fc28.x86_64                                                                                                      
    path: /usr/libexec/crio/conmon                                                                                                                       
    version: 'conmon version 1.11.1, commit: 33d1a0b346b45e1c5fd2cfd0ef27ce8da93ef1d0'                                                                   
  MemFree: 7771770880
  MemTotal: 33667186688
  OCIRuntime:
    package: runc-1.0.0-46.dev.gitb4e2ecb.fc28.x86_64
    path: /usr/bin/runc
    version: 'runc version spec: 1.0.0'
  SwapFree: 0
  SwapTotal: 0
  arch: amd64
  cpus: 8
  hostname: podhost.localdomain
  kernel: 4.17.11-200.fc28.x86_64
  os: linux
  uptime: 130h 59m 15.98s (Approximately 5.42 days)
insecure registries:
  registries: []
registries:
  registries:
  - docker.io
  - registry.fedoraproject.org
  - quay.io
  - registry.access.redhat.com
  - registry.centos.org
store:
  ContainerStore:
    number: 6
  GraphDriverName: overlay
  GraphOptions:
  - overlay.override_kernel_check=true
  GraphRoot: /var/lib/containers/storage
  GraphStatus:
    Backing Filesystem: btrfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
  ImageStore:
    number: 3
  RunRoot: /var/run/containers/storage
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
```
# rpm -q runc oci-systemd-hook conmon
runc-1.0.0-46.dev.gitb4e2ecb.fc28.x86_64
oci-systemd-hook-0.1.17-3.gitbd86a79.fc28.x86_64
conmon-1.11.1-1.git1759204.fc28.x86_64
```

**Comment by mheon** [comment]
Need a newer conmon - packaged version must be too old

**Comment by mheon** [comment]
@rhatdan @lsm5 Looks like F28 needs a newer conmon?

**Comment by aalba6675** [comment]
@mheon podman >= 0.8.1 packaging no longer contains its internal /usr/lib/podman/conmon and uses separate /usr/libexec/crio/conmon, right?

**Comment by rhatdan** [comment]
Should be fixed in podman-0.8.2.1-1.gitf38eb4f which is in release updates.
